src/components/GameItem.js

```diff
@@ -4,26 +4,49 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickedItem: false,
+    };
+  }
+  
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+// learned this through ada-students-ports project 
+  markItemClick = () => {
+    this.setState({
+      clickedItem: true,
+    });
+    this.props.type === 'litter' && this.props.onItemClickedCallback(this.props.index);
```

Comment:
Hm, what does this line do? This is an expression that would just evaluate to `true` or `false` ... 

It's important that you call `this.props.onItemClickedCallback(this.props.index)`, and this expression just happens to do that, but I don't think you need the `this.props.type === `litter' &&` part